cuesubmit/plugins/blender/OpenCue-Blender/Setup.py

```diff
@@ -0,0 +1,117 @@
+#  Copyright Contributors to the OpenCue Project
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+import shutil
+import sys
+import subprocess
+import os
+import platform
+import bpy
+
+blender_dependencies_directory = "lib/python3.10/site-packages"
+blender_dependencies_path = os.path.join(sys.prefix, blender_dependencies_directory)
+opencue_blender_path = os.path.dirname(__file__)
+
+pyoutline_path = os.path.join(opencue_blender_path, "dependencies", "outline")
+filesequence_path = os.path.join(opencue_blender_path, "dependencies", "FileSequence")
+opencue_path = os.path.join(opencue_blender_path, "dependencies", "opencue")
+
+pyoutline_directory = "outline"
+opencue_imported_directory = "opencue"
+filesequence_imported_directory = "FileSequence"
+pyoutline_directory_path = os.path.join(sys.prefix, blender_dependencies_directory, pyoutline_directory)
+opencue_directory_path = os.path.join(sys.prefix, blender_dependencies_directory, opencue_imported_directory)
+filesequence_directory_path = os.path.join(sys.prefix, blender_dependencies_directory, filesequence_imported_directory)
+
+
+def isWindows():
+    """Checks if host OS is Windows"""
+    return os.name == 'nt'
+
+
+def isMacOS():
+    """Checks if host OS is macOS"""
+    return os.name == 'posix' and platform.system() == "Darwin"
+
+
+def isLinux():
+    """Checks if host OS is Linux"""
+    return os.name == 'posix' and platform.system() == "Linux"
```

Comment:
Please use `platform.system()` to get the correct OS similar to the way rqmachine.py does it.